packages/@webex/plugin-meetings/src/reconnection-manager/index.ts

```diff
@@ -385,6 +385,29 @@ export default class ReconnectionManager {
       'ReconnectionManager:index#executeReconnection --> Attempting to reconnect to meeting.'
     );
 
+    const wasSharing = this.meeting.shareStatus === SHARE_STATUS.LOCAL_SHARE_ACTIVE;
+
+    if (wasSharing) {
+      // Stop the share streams if user tried to reconnect
+      this.meeting.setLocalShareTrack(null);
```

Comment:
this block of code is exactly the same like the one in rejoinMeeting() except for the reason, so it would be nicer to extract it into a private helper method

Reply:
Yeah I'll create a private function called "stopLocalShareTrack".